src/main/java/com/github/distributedtracing/ContinuationLocalExecutorService.java

```diff
@@ -0,0 +1,115 @@
+package com.github.distributedtracing;
```

Comment:
There are two flavors of propagating executors. One which propagates the 'current' state and one which propagates a specific 'saved' state. Should be able to support both flavors here.